### DISCUSSION TITLE
Does Qdrant v1.10 currently support ColBERT? If it does, please update the documentation about ColBERT.

### DISCUSSION THREAD

**Discussion by rfvygv** [qdrant/fastembed#322] [Q&A] [answered]
And the fastembed used by qdrant-client python sdk is not the latest one.

**Comment by joein** [comment, marked as answer]
Colbert is supported as Qdrant v1.10 - https://qdrant.tech/documentation/concepts/vectors/?q=multivec#multivectors
FastEmbed version used in qdrant-client will be updated this week, however, support for colbert and images in qdrant-client will probably be delivered a little bit later (new `query_points` endpoint will be extended to support them)

**Reply by rfvygv** [reply]
OK, thanks for your reply.
